bullet/bullet_admin/templates/bullet_admin/generic/form.html

```diff
@@ -11,7 +11,7 @@
             {% endslot %}
         {% endaheader %}
 
-        <form method="post">
+        <form method="post" enctype="multipart/form-data">
```

Comment:
I would not make this the default, rather add bool `form_multipart` or something to our `GenericForm` and add this on-demand?